.github/workflows/skms.yaml

```diff
@@ -0,0 +1,83 @@
+name: Create CMR in SKMS
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - main
+
+jobs:
+  create-cmr:
+    # Only run this workflow on pull requests that have merged and not manually closed by user
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      # Runs a single command using the runners shell for shell validation
+      - name: Run a one-line script
+        run: echo Starting CMR Action...
+
+      # Run command for creating CMR in SKMS system
+      - run: |
+          start_date=$(TZ=":America/Los_Angeles" date '+%Y-%m-%d %H:%M')
+          DEFAULT_SKMS_URL='api.skms.adobe.com'
+
+          function skms_request() {
+            local username=$1
+            local passkey=$2
+            local object=$3
+            local method=$4
+            local method_params=$5
+            local url=$6
+
+            if [ -z "$username" ]; then
```

Comment:
If you handle the errors as below, you don't need these "if"s

Reply:
see my comment above.